tests/test_extract_speakers.py

```diff
@@ -28,7 +29,7 @@ def test_extract_speakers_gpil(self):
         expected = [
             Speaker(name="Moderator", firm=None),
             Speaker(name="Ankit Toshniwal", firm="Go India Advisors"),
-            Speaker(name="Dinesh Gandhi", firm="GPIL"),
+            Speaker(name="Dinesh Gandhi", firm="Director"),
```

Comment:
Firm must not have designation. Maybe we have it for now as the algorithm is not perfect yet. If feasible we can add a separate designation field but our focus is first to map people with firms.

Reply:
This is something I need to work on